segmentation_labeling_app/transforms/array_utils.py

```diff
@@ -44,6 +44,89 @@ def content_boundary_2d(arr: Union[np.ndarray, coo_matrix]) -> np.ndarray:
     return top_bound, bot_bound, left_bound, right_bound
 
 
+def content_extents(
+        arr: Union[np.ndarray, coo_matrix],
+        shape: Tuple[int, int],
+        target_shape: Tuple[int, int] = None):
+    """return the bounding box of size shape. Intended to be applied to
+    a target data source, for example a set of video frames.
+
+    Parameters
+    ----------
+    arr: A 2d np.ndarray or coo_matrix. Other formats are also possible
+        (csc_matrix, etc.) as long as they have the toarray() method to
+         convert them to np.ndarray.
+    shape: (Tuple[int,int]) Desired final shape after padding is applied.
+        If smaller than the input array, will return the input array
+        without any changes.
+    target_shape: (Tuple[int, int]) Extent of array to be indexed. If None
+        padding will still handle top and left, but bottom and right padding
+        will always be zero
+
+    Returns
+    -------
+    indexing_bounds: tuple
+        4-tuple of row/column boundaries
+    pad_width: tuple(tuple, tuple)
+        to be passed into numpy.pad as pad_width
+    target_shape: (Tuple[int, int]) Extent of array to be indexed. If None
+        padding will still handle top and left, but bottom and right padding
+        will always be zero
+
+    Returns
```

Comment:
Duplicated Returns section of your docstring here

Reply:
fixed.